tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/Util.java

```diff
@@ -19,150 +19,15 @@
-            }
-        });
-
-        if (asyncTermination[0] && !paths.containsKey(STATUS) && !paths.containsKey(FORGET)) {
-            LOGGER.log(Level.WARNING, "LRA participant class "
-                    + compensatorClass + " with asynchronous termination but no @Status or @Forget annotations");
-            throw new GenericLRAException(null, Response.Status.BAD_REQUEST.getStatusCode(),
-                    "LRA participant class with asynchronous termination but no @Status or @Forget annotations", null);
-        }
-
-        StringBuilder linkHeaderValue = new StringBuilder();
-
-        if (paths.size() != 0) {
-            paths.forEach((k, v) -> makeLink(linkHeaderValue, null, k, v));
-            paths.put("Link", linkHeaderValue.toString());
-        }
-
-        return paths;
-    }
-
-    private static StringBuilder makeLink(StringBuilder b, String uriPrefix, String key, String value) {
-
-        if (value == null) {
-            return b;
-        }
-
-        String terminationUri = uriPrefix == null ? value : String.format("%s%s", uriPrefix, value);
-        Link link =  Link.fromUri(terminationUri).title(key + " URI").rel(key).type(MediaType.TEXT_PLAIN).build();
-
-        if (b.length() != 0) {
-            b.append(',');
-        }
-
-        return b.append(link);
-    }
-
-    /**
-     * Providing information if method is defined to be completed asynchronously.
-     * This means that {@link Suspended} annotation is available amongst the method parameters
-     * while the method is annotated with {@link Complete} or {@link Compensate}.
-     *
-     * @param method  method to be checked for async completion
-     * @return  true if method is to complete asynchronously, false if synchronously
-     */
-    public static boolean isAsyncCompletion(Method method) {
-        if (method.isAnnotationPresent(Complete.class) || method.isAnnotationPresent(Compensate.class)) {
-            for (Annotation[] ann : method.getParameterAnnotations()) {
-                for (Annotation an : ann) {
-                    if (Suspended.class.getName().equals(an.annotationType().getName())) {
-                        LOGGER.log(Level.WARNING, "JAX-RS @Suspended annotation is untested");
-                        return true;
-                    }
-                }
-            }
-        }
-
-        return false;
-    }
-
     /**
      * Adjusting the value by factor. It means it multiplies the value with factor
      * and returns the rounded (to up) number.
      *
-     * @param value  value to be adjusted
-     * @param factor  factor for adjusting value
-     * @return  adjusted value by factor
-     */
-    public static int adjust(int value, double factor) {
-        return Math.toIntExact(adjust((long) value, factor));
-    }
-
-    /**
-     * see {@link Util#adjust(int, double)}
-
      * @param value  value to be adjusted
```

Comment:
The method `adjust` is only used in one place TckTestBase. Can you move the method into that file and then delete Util.java

Reply:
@mmusgrov I was checking that when I was reviewing the code and I was not able to find such place. Can you please elaborate where is it?